src/args.rs

```diff
@@ -190,4 +194,23 @@ impl Args {
     ) -> IterBridge<WalkParallelIterator> {
         into_parallel_iterator(self.get_project_file_walker(), non_fatal_errors)
     }
+
+    pub(crate) fn get_loaded_filter(&self) -> Result<Option<Arc<Filterer>>, Error> {
```

Comment:
These were "smelling like `Args` methods" so moved them here

These both have some cost associated with them so I guess they shouldn't be thought of as "idempotent" or whatever - maybe the "loaded" in the method names gives that clue?